sound/soc/sof/ipc.c

```diff
@@ -591,7 +591,14 @@ static void ipc_stream_message(struct snd_sof_dev *sdev, u32 msg_cmd)
 
 	switch (msg_type) {
 	case SOF_IPC_STREAM_POSITION:
-		ipc_period_elapsed(sdev, msg_id);
+		if (!sdev->ops->pcm_pointer ||
+		    IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_FORCE_IPC_POSITION)) {
```

Comment:
can you make this #if like the others for consistency

Reply:
@keyonjie is this line correct, should this be !IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_FORCE_IPC_POSITION)?

Reply:
@ranj063 we handle ipc and run ipc_elapse here when force_ipc_position is configured, or we don't have pcm_pointer callback, so this line is correct.

Reply:
@keyonjie makes sense. I misunderstood the config to be to force no ipc update. Thanks!

Reply:
@lgirdwood , if use #if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_FORCE_IPC_POSITION) here, the logic may looks more ugly here?